store/mockstore/mocktikv/mock_tikv_test.go

```diff
@@ -655,6 +657,30 @@ func (s *testMVCCLevelDB) TestErrors(c *C) {
 	c.Assert((&ErrConflict{}).Error(), Equals, "write conflict")
 }
 
+func (s *testMVCCLevelDB) TestCheckTxnStatus(c *C) {
+	s.mustPrewriteWithTTLOK(c, putMutations("pk", "val"), "pk", 5, 666)
+
+	ttl, commitTS, err := s.store.CheckTxnStatus([]byte("pk"), 5, 0, 666)
+	c.Assert(err, IsNil)
+	c.Assert(ttl, Equals, uint64(666))
+	c.Assert(commitTS, Equals, uint64(0))
+
+	s.mustCommitOK(c, [][]byte{[]byte("pk")}, 5, 30)
+
+	ttl, commitTS, err = s.store.CheckTxnStatus([]byte("pk"), 5, 0, 666)
+	c.Assert(err, IsNil)
+	c.Assert(ttl, Equals, uint64(0))
+	c.Assert(commitTS, Equals, uint64(30))
+
+	s.mustPrewriteWithTTLOK(c, putMutations("pk1", "val"), "pk1", 5, 666)
+	s.mustRollbackOK(c, [][]byte{[]byte("pk1")}, 5)
+
+	ttl, commitTS, err = s.store.CheckTxnStatus([]byte("pk1"), 5, 0, 666)
+	c.Assert(err, IsNil)
+	c.Assert(ttl, Equals, uint64(0))
+	c.Assert(commitTS, Equals, uint64(0))
```

Comment:
Add test for `minCommitTS` logic?

Reply:
The minCommitTS is not returned in the CheckTxnStatus response, so we can't check it here.
Anyway, in the following PR we'll cover the `minCommitTS`
@crazycs520